.github/ISSUE_TEMPLATE/new_snippet.md

```diff
@@ -0,0 +1,15 @@
+---
+name: New snippet
+about: New snippet template
+title: 'New snippet: ...'
+labels: 'new snippets'
+---
+
+## Description
+
+## Snippet preview
+
+## Checklist before requesting a review
+
+* [ ] Have you checked to ensure there aren't other open [Issues](../../../issues) for the same update/change?
+* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
```

Comment:
Let's rephrase this

Let's add a section, name `Explanation (optional)`

and a question asking Is this snippet similar to any of the existing snippets in the project or issues.